streams/test-utils/src/main/java/org/apache/kafka/streams/test/internal/KeyValueStoreFacade.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.streams.internals;
+package org.apache.kafka.streams.test.internal;
```

Comment:
It's not great that we use `internal` here and `internals` everywhere else.

Reply:
Yes, that's exactly right. I agree it's not great. Maybe a better alternative is to just not try to mark these classes as "internal". It should be pretty hard to get confused about their usage, and apparently we have to include them in the Javadoc anyway, so it's probably fine to just move them up a level and drop `internal`